### PR TITLE
tour_summary has been renamed summary in API

### PR DESCRIPTION
**Pull request by florealcab**
So I changed it in Tour model.

### CODE REVIEW

**Comment by florealcab** [comment]
I also add the way type wt#trail (without a number) which exists now. I don't know if the numbered ones still exists so I kept them. I tested with my 80 planned routes, no error no so it sounds good :)

File: kompy/constants/way_type.py [hunk 1]
```diff
@@ -19,6 +19,7 @@ class PossibleWayType:
     TRAIL_D3: Final[str] = 'wt#trail_d3'
     TRAIL_D2: Final[str] = 'wt#trail_d2'
     TRAIL_D1: Final[str] = 'wt#trail_d1'
+    TRAIL: Final[str] = 'wt#trail'
```
Comment (Tsadoq) [inline kompy/constants/way_type.py:22]: thanks for the update!

File: kompy/tour.py [hunk 2]
```diff
@@ -153,7 +153,7 @@ def __init__(
         self.tour_information = self._create_tour_information(
             tour_information_array=tour['tour_information'],
         ) if 'tour_information' in tour else None
-        self.tour_summary = self._create_tour_summary(tour['tour_summary']) if 'tour_summary' in tour else None
+        self.summary = self._create_tour_summary(tour['summary']) if 'summary' in tour else None
```
Comment (Tsadoq) [inline kompy/tour.py:156]: good to update this to follow the APIs

**Review by Tsadoq — APPROVED** [review]

**Comment by Tsadoq** [comment]
@florealcab could you check that you change is not the reason the tests are failing?

**Comment by florealcab** [comment]
> @florealcab could you check that you change is not the reason the tests are failing?

I check it was not my fault, another way type is missing. It sounds strange as the way type comes from a json from this repository so the tests should fail before my changes.

I tested with the same tour from komoot and the way type "wt#mountain_hiking_path" really exists. So I add it too and now the tests pass :)

**Comment by florealcab** [comment]
Hi! I am working on a new project, and I am adding new functions to this package, like updating title of a tour. I need it for my project, so I want to know if you still maintain this project. In that case I can propose MR for each feature and maybe you can update package in this repo and in pipy. I can also make a fork and cite your project but it is not the best option if this repo is still active to have a high number of projects for the same thing...

**Comment by florealcab** [comment]
I recreate a PR as I have to change my branche to PR.

**Pull request closed without merging**